### PR TITLE
UA-I update /kubernetes page

### LINKED ISSUE #5042: [UA-I] Update /kubernetes page
[copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Update /kubernetes page for UA-I
- NOTE: the copy doc has an outstanding question for @Khutch25 but I have temp copy in place

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/kubernetes](http://0.0.0.0:8001/kubernetes)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#)

## Issue / Card

Fixes #5042

### CODE REVIEW

File: templates/kubernetes/index.html [hunk 1]
```diff
@@ -50,270 +56,286 @@ <h2 class="p-heading--four">Charmed Kubernetes</h2>
...
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">Extensible</h3>
+              <p class="p-matrix__desc">Easy integration of standard open source logging, monitoring, alerting, storage, networking and container runtimes. Consulting for deeper customisation is available.</p>
+            </div>
+          </li>
+          <li class="p-matrix__item">
+            <div class="p-matrix__content">
+              <h3 class="p-matrix__title p-heading--four">Fully managed</h3>
+              <p class="p-matrix__desc">At your request, Canonical’s remote operations team can build and operate Kubernetes for you, on premises or on your choice of public cloud. We will transfer control of the cluster to your ops team any time you want to take over</p>
```
Comment (anthonydillon) [inline templates/kubernetes/index.html]: Should be "on-premises"

Comment (anthonydillon) [reply templates/kubernetes/index.html]: Should also have a full stop

File: templates/kubernetes/index.html [hunk 2]
```diff
@@ -50,270 +56,286 @@ <h2 class="p-heading--four">Charmed Kubernetes</h2>
...
       </ul>
       <p>Deploy, scale and upgrade Kubernetes clusters across multiple physical or virtual machines with Ubuntu Kubernetes. Ubuntu Kubernetes includes long-term support, operations automation capabilities and Graylog, Prometheus and Grafana integration.</p>
 
       <p><a href="/kubernetes/install">Install instructions&nbsp;&rsaquo;</a></p>
       <p><a href="/kubernetes/managed">Fully managed Kubernetes options&nbsp;&rsaquo;</a>
-      <p><a href="{{ ASSET_SERVER_URL }}223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Download the Services datasheet', 'eventLabel' : 'Download the Services datasheet' : undefined });">Charmed Kubernetes datasheet&nbsp;&rsaquo;</a></p>
-    </div>
+        <p><a href="{{ ASSET_SERVER_URL }}223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Download the Services datasheet', 'eventLabel' : 'Download the Services datasheet' : undefined });">Charmed Kubernetes datasheet&nbsp;&rsaquo;</a></p>
```
[the remark this answers is not in the document]
Comment (pmahnke) [reply templates/kubernetes/index.html]: no, not for a pdf

**Comment by Khutch25** [comment]
> ## Done
> * Update /kubernetes page for UA-I
> * NOTE: the copy doc has an outstanding question for @Khutch25 but I have temp copy in place
> 
> ## QA
> * Check out this feature branch
> * Run the site using the command `./run serve`
> * View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes
> * Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
> * Compare to the [copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#)
> 
> ## Issue / Card
> Fixes #5042

@pmahnke Just responded, thank you!

**Pull request merged**